Content.Server/Chat/SuicideSystem.cs

```diff
@@ -123,19 +131,42 @@ private bool EnvironmentSuicideHandler(EntityUid victim, SuicideEvent suicideEve
             return false;
         }
 
-        private void ApplyDeath(EntityUid target, SuicideKind kind)
+        private void ApplyDeath(EntityUid target, string? kind, DamageSpecifier? damage)
         {
-            if (kind == SuicideKind.Special)
-                return;
+            ProtoId<DamageTypePrototype> fallback = "Blunt";
+            FixedPoint2 lethalAmountOfDamage = 200;
+            if (TryComp<DamageableComponent>(target, out var damagable) && TryComp<MobThresholdsComponent>(target, out var thresholds))
```

Comment:
I would check if the target has these two comps earlier and just return if they don't have it. You can't apply LethalDamage to something that can't be damaged after all.

`
if (!TryComp<DamageableComp> || !TryComp<MobThresholdComp>)
return;
`